File: packages/rook_sdk_apple_health/lib/src/data/mapper/environment_mappers.dart
import 'package:rook_sdk_apple_health/src/data/proto/protos.pbenum.dart';
import 'package:rook_sdk_core/rook_sdk_core.dart';

extension RookEnvironmentMappers on RookEnvironment {
  EnvironmentProto toProto() {
    return switch (this) {
      RookEnvironment.sandbox => EnvironmentProto.SANDBOX,
      _ => EnvironmentProto.PRODUCTION,
    };
  }
}

File: packages/rook_sdk_apple_health/ios/Classes/Mapper/EnvironmentMappers.swift
//
//  RookEnvironmentMappers.swift
//  Pods-Runner
//
//  Created by Daniel Nolasco on 13/10/23.
//

import Foundation
import RookSDK

extension EnvironmentProto {
 
    func toDomain() -> RookEnvironment {
        if self == EnvironmentProto.sandbox {
            return RookEnvironment.sandbox
        } else {
            return RookEnvironment.production
        }
    }
}
